lib/rules/jsx-one-expression-per-line.js

```diff
@@ -65,6 +65,11 @@ module.exports = {
         return;
       }
 
+      if (options.allow === 'non-jsx'
+        && !children.find((child) => (child.type === 'JSXFragment' || child.type === 'JSXElement'))) {
```

Comment:
```suggestion
      if (
        options.allow === 'non-jsx'
        && !children.find((child) => (child.type === 'JSXFragment' || child.type === 'JSXElement'))
      ) {
```

can we add some test coverage for something like `{(<div />)}` and `{(() => <div />)()}`?

Reply:
Sure, once I put my kid to sleep!

Reply:
No rush :-)